### PR TITLE
enabled security tests in multicluster env

### PR DESCRIPTION
**Pull request by aryan16**
**Enabled Security tests in mutlicluster env**
TestIncludes - 
1) mtlsk8sca
2) https_jwt
3) sds_ingress_k8sca
4) file_based_tls_origination
5) sds_ingress/quic

### CODE REVIEW

**Review by stevenctl — APPROVED** [review]

**Review by zirain — APPROVED** [review]

File: tests/integration/security/https_jwt/main_test.go [hunk 1]
```diff
@@ -46,6 +47,8 @@ func loadCert(filename string) (string, error) {
 func TestMain(m *testing.M) {
 	framework.
 		NewSuite(m).
+		Label(label.CustomSetup).
+		Label("CustomSetup").
```
Comment (howardjohn) [inline tests/integration/security/https_jwt/main_test.go:51]: nit: remove?

**Review by howardjohn — APPROVED** [review]

**Comment by nmittler** [comment]
@aryan16 I'm seeing issues now on GKE 1.21 with alpha features disabled. Specifically the error is:

```
✘ Ingress gateways encountered an error: failed to update resource with server-side apply for obj Service/istio-system/istio-ingressgateway: Service "istio-ingressgateway" is invalid: spec.ports: Invalid value: []core.ServicePort{core.ServicePort{Name:"status-port", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15021, TargetPort:intstr.IntOrString{Type:0, IntVal:15021, StrVal:""}, NodePort:0}, core.ServicePort{Name:"http2", Protocol:"TCP", AppProtocol:(*string)(nil), Port:80, TargetPort:intstr.IntOrString{Type:0, IntVal:8080, StrVal:""}, NodePort:0}, core.ServicePort{Name:"https", Protocol:"TCP", AppProtocol:(*string)(nil), Port:443, TargetPort:intstr.IntOrString{Type:0, IntVal:8443, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15443, TargetPort:intstr.IntOrString{Type:0, IntVal:15443, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls-istiod", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15012, TargetPort:intstr.IntOrString{Type:0, IntVal:15012, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls-webhook", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15017, TargetPort:intstr.IntOrString{Type:0, IntVal:15017, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tcp", Protocol:"TCP", AppProtocol:(*string)(nil), Port:31400, TargetPort:intstr.IntOrString{Type:0, IntVal:31400, StrVal:""}, NodePort:0}, core.ServicePort{Name:"http3", Protocol:"UDP", AppProtocol:(*string)(nil), Port:443, TargetPort:intstr.IntOrString{Type:0, IntVal:8443, StrVal:""}, NodePort:0}}: may not contain more than 1 protocol when type is 'LoadBalancer'
```

Any idea if this might be related?

**Comment by nmittler** [comment]
@aryan16 Just confirmed that installation succeeds for me when I rollback this change locally. Can you rollback while you investigate?

**Comment by DSP-doublehappy111** [comment]
> @aryan16 I'm seeing issues now on GKE 1.21 with alpha features disabled. Specifically the error is:
> 
> ```
> ✘ Ingress gateways encountered an error: failed to update resource with server-side apply for obj Service/istio-system/istio-ingressgateway: Service "istio-ingressgateway" is invalid: spec.ports: Invalid value: []core.ServicePort{core.ServicePort{Name:"status-port", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15021, TargetPort:intstr.IntOrString{Type:0, IntVal:15021, StrVal:""}, NodePort:0}, core.ServicePort{Name:"http2", Protocol:"TCP", AppProtocol:(*string)(nil), Port:80, TargetPort:intstr.IntOrString{Type:0, IntVal:8080, StrVal:""}, NodePort:0}, core.ServicePort{Name:"https", Protocol:"TCP", AppProtocol:(*string)(nil), Port:443, TargetPort:intstr.IntOrString{Type:0, IntVal:8443, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15443, TargetPort:intstr.IntOrString{Type:0, IntVal:15443, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls-istiod", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15012, TargetPort:intstr.IntOrString{Type:0, IntVal:15012, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls-webhook", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15017, TargetPort:intstr.IntOrString{Type:0, IntVal:15017, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tcp", Protocol:"TCP", AppProtocol:(*string)(nil), Port:31400, TargetPort:intstr.IntOrString{Type:0, IntVal:31400, StrVal:""}, NodePort:0}, core.ServicePort{Name:"http3", Protocol:"UDP", AppProtocol:(*string)(nil), Port:443, TargetPort:intstr.IntOrString{Type:0, IntVal:8443, StrVal:""}, NodePort:0}}: may not contain more than 1 protocol when type is 'LoadBalancer'
> ```
> 
> John suggested this may be related. Can you take a look?

I also had the same problem now. Is there any solution

**Comment by DSP-doublehappy111** [comment]
> @aryan16 I'm seeing issues now on GKE 1.21 with alpha features disabled. Specifically the error is:
> 
> ```
> ✘ Ingress gateways encountered an error: failed to update resource with server-side apply for obj Service/istio-system/istio-ingressgateway: Service "istio-ingressgateway" is invalid: spec.ports: Invalid value: []core.ServicePort{core.ServicePort{Name:"status-port", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15021, TargetPort:intstr.IntOrString{Type:0, IntVal:15021, StrVal:""}, NodePort:0}, core.ServicePort{Name:"http2", Protocol:"TCP", AppProtocol:(*string)(nil), Port:80, TargetPort:intstr.IntOrString{Type:0, IntVal:8080, StrVal:""}, NodePort:0}, core.ServicePort{Name:"https", Protocol:"TCP", AppProtocol:(*string)(nil), Port:443, TargetPort:intstr.IntOrString{Type:0, IntVal:8443, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15443, TargetPort:intstr.IntOrString{Type:0, IntVal:15443, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls-istiod", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15012, TargetPort:intstr.IntOrString{Type:0, IntVal:15012, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tls-webhook", Protocol:"TCP", AppProtocol:(*string)(nil), Port:15017, TargetPort:intstr.IntOrString{Type:0, IntVal:15017, StrVal:""}, NodePort:0}, core.ServicePort{Name:"tcp", Protocol:"TCP", AppProtocol:(*string)(nil), Port:31400, TargetPort:intstr.IntOrString{Type:0, IntVal:31400, StrVal:""}, NodePort:0}, core.ServicePort{Name:"http3", Protocol:"UDP", AppProtocol:(*string)(nil), Port:443, TargetPort:intstr.IntOrString{Type:0, IntVal:8443, StrVal:""}, NodePort:0}}: may not contain more than 1 protocol when type is 'LoadBalancer'
> ```
> 
> John suggested this may be related. Can you take a look?

I've address this problem due to kube-apiserver need to open the mixed protocols in service.type=loadbalancer with `--feature-gates=MixedProtocolLBService=true`.The mixed lbservice not enabled by default.

**Pull request merged**